### PR TITLE
Add an introduction for ASCornerLayoutSpec in layout2-layoutspec-types.md #trivial

### PR DESCRIPTION
**Pull request by huang-kun**
Add an introduction for ASCornerLayoutSpec in layout2-layoutspec-types.md

### CODE REVIEW

**Comment by maicki** [comment]
Hey @huang-kun I think this PR was opened accidentally. I will close it. Thanks!

**Comment by huang-kun** [comment]
@maicki  Would you add an introduction for new features in version 2.7?  [Here](https://github.com/TextureGroup/Texture/pull/1005/commits/d92d5ec15a9e8601d7d75fbcc1f51bb7f8a40ffc)

**Comment by maicki** [comment]
@huang-kun Ah sorry I apologize for closing it. There were merge conflicts in there and the title was not really describing what the PR is about, so I thought it was created accidentally. 

I will reopen the PR, can you please remove all of the merge conflicts and squash the branch into one?

Thanks!

File: Source/Layout/ASCornerLayoutSpec.h [hunk 1]
```diff
@@ -49,30 +49,50 @@ NS_ASSUME_NONNULL_BEGIN
  @param location The corner position option.
  @return An ASCornerLayoutSpec object with a given child and an layoutElement that act as corner.
  */
+<<<<<<< HEAD
```
Comment (maicki) [inline Source/Layout/ASCornerLayoutSpec.h]: Please remove all of the merge conflicts.

**Comment by garrettmoon** [comment]
@huang-kun would you mind squashing your commits or updating the author on them? A few of them look like they have a non-github author.

**Pull request closed without merging**